src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -62,12 +62,14 @@ def get_initial_value(self, submission: Submission | None = None):
 @register("payment_amount")
 class PaymentAmount(BaseStaticVariable):
     name = _("Payment amount")
-    data_type = FormVariableDataTypes.string
+    data_type = FormVariableDataTypes.float
```

Comment:
I don't know enough about this, but why was this changed from string to float? Or was this incorrectly marked as string to begin with?

Reply:
was incorrectly marked as string, since clearly floats (and `None`) were being returned :)